Cargo.toml

```diff
@@ -1,7 +1,7 @@
 [package]
 name = "rumqtt"
 description = "Mqtt client for your IOT needs"
-version = "0.31.0"
+version = "0.32.0"
```

Comment:
I am personally split here, I would rather push for a 0.31.1 tag here instead of bumping minor, but it is a grey area since it actually changes return errors available.

But since the current jumps in this library has been major rewrites between minor releases, i still think doing this is better, It is actually a bugfix in essence from my point of view

Reply:
It's a very minor (heh) bug fix yes but from a API semvar perspective, adding a new variant to an enum not marked `#[non_exhaustive]` is a minor bump.

Reply:
Your reasoning is sound and I do agree that pub enum changes are breaking api, but in this regard I feel you fix a crash and I would like to get changes out asap. 

Because the other option is to not bump the version yet, since this change doesn't really warrant it.

Reply:
I just stumbled upon this panic. Would be good get better error message.